cmd/main.go

```diff
@@ -79,8 +79,6 @@ func main() {
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
```

Comment:
Did you mean to remove these?

Reply:
Yeah.

https://github.com/kubernetes-sigs/controller-runtime/pull/2407

Controller runtime reimplemented the metrics server configuration, and given the state of this project I didn't think it was worth reconfiguring so I just removed the metrics configuration rather than reimplementing it. It wouldn't be a ton of effort to do, but I think we want to try to move toward getting rid of this thing entirely once we start swapping out rukpak.